### PR TITLE
Fixes drag handle for picker fields #3553

### LINKED ISSUE #3553: [3.6.0-alpha.2] Pages, files and users field not sortable, no drag handle
**Describe the bug**  
While pages **sections** have a drag handle, I can not sort the selected pages in a pages **field**.

**To Reproduce**  
1. Add a files field
2. Select files and try to sort them

**Expected behavior**  
A drag handle so I can sort the pages.

**Kirby Version**  
3.6.0-alpha.2

**Desktop (please complete the following information):**  
 - OS: macOS Big Sur 11.4
 - Browser: Chrome 
 - Version: 91.0.4472.114

### PR DESCRIPTION
**Pull request by afbora**
## Describe the PR

`item.sortable` is related for sections (model permissions). I've fixed the issue with checking `sortable` definition. Page/User/File fields items no need sort permission.

### Regression Fixes

- Fixed drag handle for picker fields `#3553`

## Breaking changes

None

## Related issues/ideas

- Fixes #3553 

## Ready?

- [ ] Unit tests for fixed bug/feature
- [x] In-code documentation (wherever needed)
- [x] CI checks pass

## When merging

- [x] Add to [website docs release checklist](https://github.com/getkirby/getkirby.com/pulls) (if needed)
- [x] Add changes to release notes draft in Notion

### CODE REVIEW

File: panel/src/components/Layout/Items.vue [hunk 1]
```diff
@@ -15,10 +15,10 @@
         v-for="(item, itemIndex) in items"
         :key="item.id || itemIndex"
         v-bind="item"
-        :class="{'k-draggable-item': item.sortable}"
+        :class="{'k-draggable-item': sortable && (item.sortable !== undefined ? item.sortable : true)}"
```
Comment (distantnative) [inline panel/src/components/Layout/Items.vue]:
```suggestion
        :class="{'k-draggable-item': sortable && (item.sortable || true)}"
```

Would this work as well?

Comment (afbora) [reply panel/src/components/Layout/Items.vue]: I thought so too at first but `false || true` returns `true`. We need `item.sortable ?? true` but our panel still doesn't support:
 https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator

Comment (distantnative) [reply panel/src/components/Layout/Items.vue]: Ahh of course, you are right!

Comment (afbora) [reply panel/src/components/Layout/Items.vue]: Why `Nullish coalescing operator` doesn't working. I can test on JSFiddle but in our panel throwing error on `npm run build`. Do you have any idea?

Comment (bastianallgeier) [reply panel/src/components/Layout/Items.vue]: We should fix this in way that item.sortable is already a boolean and the check is not needed.

Comment (afbora) [reply panel/src/components/Layout/Items.vue]: @bastianallgeier `item.sortable` is `boolean` on sections but `undefined` on pages/files/users field.

Comment (bastianallgeier) [reply panel/src/components/Layout/Items.vue]: Then let's fix it on the PHP side

Comment (afbora) [reply panel/src/components/Layout/Items.vue]: With setting `item.sortable = true` as default on PHP for picker fields?

Comment (afbora) [reply panel/src/components/Layout/Items.vue]: @bastianallgeier I've push commit https://github.com/getkirby/kirby/pull/3557/commits/6ec420d3a2cb58a00bcaee10f6130591c57617b6. Like that?

**Comment by bastianallgeier** [comment]
@afbora yep, that's great!

**Review by bastianallgeier — APPROVED** [review]

**Pull request merged**